File: mobile-client/android/app/src/main/java/com/smartresponsor/mobile/app/order/OrderOwnedRouteMap.kt
package com.smartresponsor.mobile.app.order
class OrderOwnedRouteMap {
    fun ownedFlows(): List<OrderOwnedFlow> = listOf(OrderOwnedFlow.SHIPMENT)
    fun embeddedCapabilities(): List<OrderEmbeddedCapability> = listOf(OrderEmbeddedCapability.TAXATION)
}

File: mobile-client/ios/App/Order/OrderOwnedRouteMap.swift
import Foundation

// Marketing America Corp. Oleksandr Tishchenko
// Canonical route ownership map for order-owned flows.
public struct OrderOwnedRouteMap: Sendable {
    public init() {}

    public func ownedFlows() -> [OrderOwnedFlow] {
        [.shipment, .taxation]
    }
}
